include/magic_enum.hpp

```diff
@@ -342,8 +369,14 @@ inline constexpr bool is_enum_v = std::is_enum_v<T> && std::is_same_v<T, std::de
 template <typename E>
 constexpr auto n() noexcept {
   static_assert(is_enum_v<E>, "magic_enum::detail::n requires enum type.");
+  using std::holds_alternative;
```

Comment:
this `holds_alternative` will work with `MAGIC_ENUM_USING_ALIAS_VARIANT` ?
I think it safer to check the `variant::index` constexpr function result.

Reply:
make sense,yes